scenes/game/local_game.gd

```diff
@@ -5283,6 +5291,62 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			if (other_start < performing_start and new_location > performing_start) or (other_start > performing_start and new_location < performing_start):
 				local_conditions.pulled_past = true
 			_append_log_full(Enums.LogType.LogType_CharacterMovement, opposing_player, "is pulled %s, moving from space %s to %s." % [str(effect['amount']), str(previous_location), str(new_location)])
+		"pull_any_number_of_spaces_and_gain_power":
+			decision_info.clear()
+			decision_info.type = Enums.DecisionType.DecisionType_ChooseArenaLocationForEffect
+			decision_info.player = performing_player.my_id
+			decision_info.choice_card_id = card_id
+			decision_info.effect_type = "pull_to_space_and_gain_power"
+			decision_info.choice = []
+			decision_info.extra_info = ""
+			decision_info.limitation = []
+
+			decision_info.limitation.append(0)
+			decision_info.choice.append({ "effect_type": "pass" })
+			
+			var player_location = performing_player.arena_location
+			var opponent_location = opposing_player.arena_location
+			var nowhere_to_pull = true
+			for i in range(MinArenaLocation, MaxArenaLocation+1):
+				if opposing_player.is_overlapping_opponent(i):
+					continue
+				if opponent_location == i:
+					continue
+				if player_location < opponent_location and i > opponent_location:
+					continue
+				if player_location > opponent_location and i < opponent_location:
+					continue
+				decision_info.limitation.append(i)
+				decision_info.choice.append({
+					"effect_type": "pull_to_space_and_gain_power",
+					"amount": i
+				})
+				nowhere_to_pull = false
+
+			if not nowhere_to_pull:
+				change_game_state(Enums.GameState.GameState_PlayerDecision)
+				events += [create_event(Enums.EventType.EventType_ChooseArenaLocationForEffect, performing_player.my_id, 0)]
+		"pull_to_space_and_gain_power":
+			var space = effect['amount']
+			var previous_location = opposing_player.arena_location 
+			var distance = opposing_player.movement_distance_between(space, previous_location)
+			if space == previous_location:
+				# This effect should only be called with an actual attempt to pull.
+				assert(false)
+			elif space < previous_location and performing_player.arena_location < previous_location \
+			or space > previous_location and performing_player.arena_location > previous_location:
+				events += performing_player.pull(distance)
+				var new_location = opposing_player.arena_location
+				var pull_amount = opposing_player.movement_distance_between(previous_location, new_location)
+				local_conditions.pull_amount = pull_amount
```

Comment:
I don't really understand why you added this to LocalConditions.
That's more intended for if an effect has and/bonus_effect and there's a condition that actually reads it.
Nothing seems to actually read this? Unless I'm missing something.

Reply:
If you were actually combining this effect with like push_or_pull_to_any_space and had an "and" effect that was like set_strike_x to the pull amount and then powerup strike_x or something, then I could see this being useful.
As is though, it seems pointless.

Reply:
Good catch, leftover from when I was trying to reuse the effect between ultras instead of how I settled on doing them. Removing it.